services/worker/src/worker/job_runners/split/first_rows_from_parquet.py

```diff
@@ -10,11 +10,13 @@
     PROCESSING_STEP_SPLIT_FIRST_ROWS_FROM_STREAMING_VERSION,
 )
 from libcommon.exceptions import (
+    ParquetResponseEmptyError,
     RowsPostProcessingError,
     TooBigContentError,
     TooManyColumnsError,
 )
 from libcommon.parquet_utils import Indexer, TooBigRows
+from libcommon.parquet_utils import ParquetResponseEmptyError as EmptyError
```

Comment:
maybe rename ParquetResponseEmptyError in libcommon.parquet_utils to prevent future confusions?

Reply:
Or directly raise libcommon.exceptions.ParquetResponseEmptyError there? It would remove the need to put the following code in a try/except structure

Reply:
> Or directly raise libcommon.exceptions.ParquetResponseEmptyError there? It would remove the need to put the following code in a try/except structure

I thought the same but, I was not sure if it is appropriate to return a `CacheableError` in parquet_utils that could be exposed in /rows.


> maybe rename ParquetResponseEmptyError in libcommon.parquet_utils to prevent future confusions?

I like more this approach

Reply:
> I thought the same but, I was not sure if it is appropriate to return a CacheableError in parquet_utils that could be exposed in /rows.

You're right